src/Model/Contact.php

```diff
@@ -1245,10 +1245,10 @@ public static function isHidden($cid)
 	/**
 	 * Returns posts from a given contact url
 	 *
-	 * @param string $contact_url Contact URL
-	 * @param bool   $thread_mode
-	 * @param int    $update
-	 * @param int    $parent
+	 * @param string  $contact_url Contact URL
+	 * @param bool    $thread_mode
+	 * @param integer $update      Update mode 
+	 * @param int     $parent      Parent ID for the update mode
```

Comment:
What table is it an ID from? Can there be an update mode without a parent id?

Reply:
Yes. This is the case for the regular background update. The `$parent` is only set when you comment on something.